pages/common/dolt.md

```diff
@@ -1,6 +1,6 @@
 # dolt
 
-> Dolt is a SQL database that you can fork, clone, branch, merge, push and pull just like a Git repository.
+> Dolt is an SQL database that you can fork, clone, branch, merge, push and pull just like a Git repository.
```

Comment:
This is tricky! What if you read it (as I do) as "sequel," rather than "S-Q-L"?

I wonder if the latter _is_ more common tho 🤔

Reply:
Valid concern. 🤔 

I'd lean on "an" myself.
LanguageTool reports this should be "An SQL", but that's ofc going to be on the assumption everyone uses S-Q-L.

> The original standard declared that the official pronunciation for "SQL" was an initialism: /ˌɛsˌkjuːˈɛl/ ("ess cue el").[9] Regardless, many English-speaking database professionals (including Donald Chamberlin himself[31]) use the acronym-like pronunciation of /ˈsiːkwəl/ ("sequel"),[32] mirroring the language's prerelease development name, "SEQUEL".[13][14][31]
> \- https://en.wikipedia.org/wiki/SQL#Standardization_history

Based on this, I'd vouch "an SQL" - and people that chose to read it as "sequel" can just auto-correct it their head.

Reply:
Sure.  I'll go ahead and keep after Mr. Chamberlin and use "seque," but I concede the point here.